numpy/core/tests/test_scalarmath.py

```diff
@@ -911,7 +911,7 @@ def test_scalar_signed_integer_overflow(dtype, operation):
 
 @pytest.mark.parametrize("dtype", np.typecodes["UnsignedInteger"])
 @pytest.mark.xfail  # TODO: the check is quite simply missing!
```

Comment:
```suggestion
```
OK, nvm trying to let you to dig, just remove this :).  I suppose with this gone, all "new" paths should be hit.
I had just noticed the xfail was still here, so that is bad.  But of course the main (abs) change is tested.

Reply:
Sorry, this was really not the PR at all... But, I still think we can remove the xfail now?